Pokedex_ios/View/Controller/PokemonListViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ViewController.swift
+//  Pokedex_ios
+//
+//  Created by admin on 6/29/22.
+//
+
+import UIKit
+
+class PokemonListViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    lazy var viewModel = {
+        PokemonListViewModel()
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.delegate = self
+        tableView.dataSource = self
+        setBackground()
+        initViewModel()
+        
+        let uiNib = UINib(nibName: "PokemonTableViewCell", bundle: nil)
```

Comment:
any text identifier is recommended to be as a constant on top of the file or in a secondary constants file.